### PR TITLE
[WEBSOCKET-120] WebSocket 1.1 docs

### PR DESCRIPTION
**Pull request by jkonrath-postman**
See WEBSOCKET-120

### CODE REVIEW

**Review by codenirvana — CHANGES_REQUESTED** [review]

File: src/pages/docs/sending-requests/supported-api-frameworks/websocket.md [hunk 1]
```diff
@@ -76,6 +92,18 @@ In an expanded message:
 * When you hover over line numbers, caret controls (**v**) are displayed. Click them to expand or collapse blocks of the message.
 [![WebSocket message body](https://assets.postman.com/postman-docs/websocket-message-body.jpg)](https://assets.postman.com/postman-docs/websocket-message-body.jpg)
 
+### Event listening in Socket.IO
+
+In Socket.IO, you have the ability to listen to specific events. The **Messages** pane will only display the events for which you've added listeners. Events will be color-coded according to event, to make them easier to find.
```
Comment (sowmyaraj92) [inline src/pages/docs/sending-requests/supported-api-frameworks/websocket.md:97]: "according to the event", maybe?

File: src/pages/docs/sending-requests/supported-api-frameworks/websocket.md [hunk 2]
```diff
@@ -64,7 +80,7 @@ The following are displayed for each message:
   [![WebSocket time difference](https://assets.postman.com/postman-docs/websocket-time-difference.gif)](https://assets.postman.com/postman-docs/websocket-time-difference.gif)
 * Click **>** to expand a message, or **v** to collapse it.
 * The time is displayed, as your local time.
-* If you hover over the time, an information icon is displayed. Hover over it to see the time, MIME type, and size of the message.
+* If you hover over the time of a raw SocketIO message, an information icon is displayed. Hover over it to see the time, MIME type, and size of the message.
```
Comment (sowmyaraj92) [inline src/pages/docs/sending-requests/supported-api-frameworks/websocket.md:83]: Is it "SocketIO message" or "Socket.IO message"?

File: src/pages/docs/sending-requests/supported-api-frameworks/websocket.md [hunk 3]
```diff
@@ -23,27 +18,48 @@ The WebSocket protocol provides a way to exchange data between a client and serv
...
+
+In addition to raw WebSocket connections, Postman also supports Socket.IO connections. Socket.IO is one of the most popular libraries to enable event-drive, bidirectional, real-time communication between clients and servers. It uses WebSocket as its transport layer. Many developers use Socket.IO in combination with HTTP APIs; now you can enjoy the benefits of Postman while switching between these two paradigms at will.
+
 ## Creating WebSocket requests
 
 You can create a WebSocket request from the left sidebar in Postman.
 
 1. Click  __New__ &gt; __WebSocket Request__ to open a new tab. (You can also select ⌘+N or Ctrl+N.)
   [![New Screen](https://assets.postman.com/postman-docs/new-request-websocket.jpg)](https://assets.postman.com/postman-docs/new-request-websocket.jpg)
+1. In the tab header, select either **Raw** for a raw WebSocket request, or **Socket.IO** for a Socket.IO request.
```
Comment (sowmyaraj92) [inline src/pages/docs/sending-requests/supported-api-frameworks/websocket.md:31]: Can we confirm the terminology for "tab header" here?

File: src/pages/docs/sending-requests/supported-api-frameworks/websocket.md [hunk 4]
```diff
@@ -2,18 +2,13 @@
...
 order: 38
 page_id: "websocket"
-search_keyword: "WebSocket, websockets, WebSocket requests"
+search_keyword: "WebSocket, websockets, WebSocket requests, Socket.IO"
 contextual_links:
   - type: section
     name: "Prerequisites"
   - type: link
     name: "Installing and updating"
     url: "/docs/getting-started/installation-and-updates/"
```
Comment (sowmyaraj92) [inline src/pages/docs/sending-requests/supported-api-frameworks/websocket.md:11]: Can we add a blog link for Websockets - https://blog.postman.com/postman-supports-websocket-apis/

**Review by sowmyaraj92 — COMMENTED** [review]
Looks great overall, suggested minor edits.

File: src/pages/docs/sending-requests/supported-api-frameworks/websocket.md (hunk 3 above)
Comment (jkonrath-postman) [reply src/pages/docs/sending-requests/supported-api-frameworks/websocket.md:31]: Content doesn't have a term for it in their word list, so I'm changing it to "upper left of the tab."

File: src/pages/docs/sending-requests/supported-api-frameworks/websocket.md (hunk 4 above)
Comment (jkonrath-postman) [reply src/pages/docs/sending-requests/supported-api-frameworks/websocket.md:11]: We're adding all of these as part of Claire's thing. There's already a GH issue for this, so we'll do it there.

**Pull request merged**